### PR TITLE
[Please ignore for review ] draft PR to fix version 2019 05 01 validation errors

### PR DESCRIPTION
**Pull request by leizhaoPS**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by NelsonDaniel** [comment]
@leizhaoPS you can create a draft PR using github instead of changing the title: https://github.blog/2019-02-14-introducing-draft-pull-requests/
Can you create a draft PR using this feature, and then mark it as ready when you are done?

**Comment by leizhaoPS** [comment]
Thanks for notice, I will create a draft PR instead of this

**Comment by jhendrixMSFT** [comment]
@kpajdzik can you look at the typescript generation failure?

File: specification/machinelearningservices/resource-manager/readme.go.md [hunk 1]
```diff
@@ -13,9 +13,29 @@ go:
...
+  - tag: package-2018-11-19
   - tag: package-2018-03-preview
 ```
 
+### Tag: package-2019-05-01 and go
+
+These settings apply only when `--tag=package-2019-05-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag)=='package-2018-03-preview' && $(go)
```
Comment (jhendrixMSFT) [inline specification/machinelearningservices/resource-manager/readme.go.md]: The tag is wrong, it should be `package-2019-05-01`

File: specification/machinelearningservices/resource-manager/readme.go.md [hunk 2]
```diff
@@ -13,9 +13,29 @@ go:
...
+output-folder: $(go-sdk-folder)/services/machinelearning/mgmt/2019-05-01/$(namespace)
+```
+
+### Tag: package-2018-11-19 and go
+
+These settings apply only when `--tag=package-2018-11-19 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag)=='package-2018-03-preview' && $(go)
```
Comment (jhendrixMSFT) [inline specification/machinelearningservices/resource-manager/readme.go.md]: The tag is wrong, it should be `package-2018-11-19`

Comment (leizhaoPS) [reply specification/machinelearningservices/resource-manager/readme.go.md]: Great Thanks! The issue fixed now and it is the root cause!!

**Comment by leizhaoPS** [comment]
@kpajdzik the Typescript errors gone with the go tag missmatch issue

**Comment by leizhaoPS** [comment]
fixes updated to official PR and closed this one

**Pull request closed without merging**